{"uuid": "117d7324-bd95-4bb9-852a-5c01b27e1caa", "counter": 115, "stdout": "\u001b[0;34m<192.168.38.8> (1, b'\\r\\n{\"changed\": true, \"stdout\": \"\", \"stderr\": \"python3: can\\'t open file \\'/home/vmadm/southbound/subnetting.py\\': [Errno 2] No such file or directory\", \"rc\": 2, \"cmd\": [\"python3\", \"southbound/subnetting.py\", \"192.168.100.0/24\"], \"start\": \"2024-04-21 18:45:26.471498\", \"end\": \"2024-04-21 18:45:26.505411\", \"delta\": \"0:00:00.033913\", \"failed\": true, \"msg\": \"non-zero return code\", \"invocation\": {\"module_args\": {\"_raw_params\": \"python3 southbound/subnetting.py 192.168.100.0/24\", \"_uses_shell\": false, \"stdin_add_newline\": true, \"strip_empty_ends\": true, \"argv\": null, \"chdir\": null, \"executable\": null, \"creates\": null, \"removes\": null, \"stdin\": null}}}\\r\\n', b'Shared connection to 192.168.38.8 closed.\\r\\n')\u001b[0m", "start_line": 108, "end_line": 109, "runner_ident": "fe7d54f6-3b66-4d1a-b163-2cb25ff7b586", "event": "verbose", "pid": 3564009, "created": "2024-04-21T18:45:26.561187", "parent_uuid": "52540038-3809-6217-eb16-000000000005", "event_data": {"playbook": "/home/vmadm/demo/RapidServe/southbound/deploy_vpc.yaml", "playbook_uuid": "407bd115-401b-4758-b8e0-1669a8cede46", "play": "Deploy VPC Infrastructure with network namespace", "play_uuid": "52540038-3809-6217-eb16-000000000002", "play_pattern": "hostvms", "task": "Get /30 subnet for current VPC", "task_uuid": "52540038-3809-6217-eb16-000000000005", "task_action": "command", "resolved_action": "ansible.builtin.command", "task_args": "", "task_path": "/home/vmadm/demo/RapidServe/southbound/deploy_vpc.yaml:11", "uuid": "117d7324-bd95-4bb9-852a-5c01b27e1caa"}}